docs/navigation.rb

```diff
@@ -109,6 +109,9 @@ def without_prefix(prefix)
           {
             path: 'Module-Documentation.md'
           },
+          {
+            path: 'Building-The-Docs-Locally.md'
```

Comment:
I'm not sure this information would belong in the 'basics' category

Reply:
Got an idea for where to put this? I put it in this category as this is stuff people would likely want to read when starting out, hence the basics idea since this would be key information to have on an onsite pentest, but feel free to recommend another location that is more appropriate.

Reply:
@adfoster-r7 Any update on this or where we should place it? Happy to correct things but think this is the last blocker.

Reply:
Bump

Reply:
I'm still in favor of not having this page, as it doesn't seem clear who this functionality is aimed at.

That aside, if we did want this page - I think it would live under `Other` rather than `Basics`. And I think we'd want to update the link to be `Offline documentation` - as 'building the docs locally' feels more like a technical implementation of the hypothetical user's actual requirements.

I think we'd want to sprinkle in more keywords to the markdown file too for SEO purposes, i.e. mentioning the terminology that the users would search for - i.e. offline Metasploit documentation / airgapped environments / etc. It's maybe out of scope, but you could also litter in keywords such as 'Metasploit manual' etc. We could use Google analytics to see how much traffic is going to this page to see if it's worth investing more time in.

Reply:
So if you don't mind me asking, what would be the preferred solution for this @adfoster-r7 in your eyes? As I see it right now we have a way to build local documentation and no target audience for that functionality which to me seems like a potential case of code maintenance for something that isn't being used, so its making a bit confused as to how to approach this scenario.

The only other target audience I could come up with beyond onsite pentesters is those in countries where internet quality is poor or those on limited internet connections where they can only download so much data per month who might want to save data over the long haul, however I don't know what percentage size of our users is the latter group and the former (aka pentesters) that I've spoken too seem to have not used offline documentation as much as I was initially expecting.